aws/raw-to-ami.sh

```diff
@@ -54,7 +54,7 @@ set_bucket_policies() {
 }
 EOF
 
-	aws iam create-role --role-name vmimport --assume-role-policy-document "file://${TRUST_POLICY_JSON_FILE}"
+	aws iam create-role --role-name vmimport --assume-role-policy-document "file://${TRUST_POLICY_JSON_FILE}" --region ${REGION}
```

Comment:
May be a better approach is to take the region specified in the argument of the raw-to-ami.sh and use it for the aws command ?
Also can we use the REGION from the `AWS_REGION` env variable (which is also a common approach) and if it's not set, then throw an error to explicitly provide the REGION option ?

Reply:
> May be a better approach is to take the region specified in the argument of the raw-to-ami.sh and use it for the aws command ?

this is what i did (or i didn't understand what you mean)

about the AWS_REGION, it's a bit tricky as it could also come from AWS_DEFAULT_REGION or from the profile configuration, but indeed maybe i can just take into account only AWS_REGION only

Reply:
sent another version